### DISCUSSION TITLE
squeezebox touch as a server

### DISCUSSION THREAD

**Discussion by Markolino1972** [community/community#38413] [Other Feature Feedback, Questions, & Ideas]
hello
first of all thank you for your wonderfull work (squeezeplay)
i tried firmware 8.0.1 for thr SBT but there is no more tinysc
so i'm using 7.5.0 to make my sbt a server for my transporter (only ethernet) i control everything with squeezeplay on M1 and thank you very much for that.
how i can tell the SBT to stop looking for update 7.7.3?
is there a way to shut down unnecessary plugins on the SBT aswell?
i've already looked on the file but my skills are not enough to spot where to do things...
thank you

**Comment by jfullstackdev** [comment]
if I'm also referring to the same software, fortunately it's on GitHub 

https://github.com/Logitech/slimserver

you can raise an issue and ask them
